README.md

```diff
@@ -39,8 +39,9 @@ for more information about ssh agents.
 
 ```js
 var fmt = require('util').format;
+var fs = require('fs');
 var http = require('http');
-var st = require('strong-tunnel');
+var st = require('../');
```

Comment:
Aha! I knew I was missing something with my copy/pasting.. trying to keep the example _in_ the README in sync with the code in `test/example.js` so that the docs are "always right"